test/filterStateProps.spec.js

```diff
@@ -0,0 +1,239 @@
+import { expect } from 'chai'
+import { createStore } from 'redux'
+import undoable, { ActionTypes } from '../src/index'
+
+describe('Undoable with filterStateProps', () => {
+  let initialStoreState = {
+    position: { // will be excluded from history
+      x: 0,
+      y: 0
+    },
+    counter: 0
+  }
```

Comment:
Just a note from another user of redux-undo: In this case, it would make sense to just compose this as two reducers and then wrap only one of them with `undoable`.

Something like:

```
const baseReducer = combineReducers({
  position: positionReducer,
  count: undoable(countReducer),
})
```

I understand this is a very simplistic example, but I think it can be extrapolated to any scenario - you can simply put the data you want to be ignored into another slice of state and call it a day. What would be the benefit of filtering like this?